app/js/lib/form_validation.js

```diff
@@ -82,29 +82,40 @@ var jQuery = require( 'jquery' ),
 		validationUrlForNonSepa: '',
 		sendFunction: null,
 		validate: function ( formValues ) {
-			var data, validationUrl;
 			if ( formValues.paymentType && formValues.paymentType !== 'BEZ' ) {
 				return {
 					status: 'OK'
 				};
 			}
+
 			if ( formValues.debitType === 'sepa' ) {
-				data = {
-					iban: formValues.iban
-				};
-				validationUrl = this.validationUrlForSepa;
-			} else {
-				data = {
-					accountNumber: formValues.accountNumber,
-					bankCode: formValues.bankCode
-				};
-				validationUrl = this.validationUrlForNonSepa;
+				if ( formValues.iban === '' ) {
+					return { status: 'INCOMPLETE' };
+				}
+
+				return this.sendFunction(
+					this.validationUrlForSepa,
+					{
+						iban: formValues.iban
+					},
+					null,
+					'json'
+				);
```

Comment:
It took me some digging to realize that the signature here is the one of `jQuery.get`. I'm quite dubious about the fact that we make web requests show up here, especially in a manner so prominent when reading the code here.